# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gcc/multilib-powerpc64le.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-12.2.0/gcc/config.gcc.vanilla	2023-01-05 22:52:10.358042459 +0100
+++ gcc-12.2.0/gcc/config.gcc	2023-01-05 22:53:30.582047017 +0100
@@ -2999,7 +2999,7 @@
 	    all) maybe_biarch=yes ;;
 	esac
 	case ${target}:${enable_targets}:${maybe_biarch} in
-	    powerpc64-* | powerpc-*:*:yes | *:*powerpc64-*:yes | *:all:yes \
+	    powerpc*:*:yes | *:*powerpc64-*:yes | *:all:yes \
 	    | powerpc64le*:*powerpcle* | powerpc64le*:*powerpc-* \
 	    | powerpcle-*:*powerpc64le*:yes)
 		if test x$cpu_is_64bit = xyes; then
